steps/test_python.sh

```diff
@@ -18,8 +18,6 @@ set -ex
 
 ROOT_DIR=$(pwd)
 
-bash onebox/stop_all.sh
```

Comment:
is it better to fix `pkill -9 -x -l openmldb` in stop_all.sh? e.g. `pgrep openmldb | kill`
stop_all.sh in script is good for retrying test